docs/java/how-to-develop-a-workflow-definition-in-java.md

```diff
@@ -50,19 +47,106 @@ public interface FileProcessingWorkflow {
 }
 ```
 
-The `@WorkflowMethod` annotation has a `name` parameter, such as `@WorkflowMethod(name = "MyWorkflowType")`.
-It can be used to denote the [Workflow Type](/docs/concepts/what-is-a-workflow-type). If not set, the Workflow Type defaults to the short name of the Workflow interface. In the example above, the Workflow Type defaults to `FileProcessingWorkflow`.
+### `@WorkflowMethod`
+
+The `@WorkflowMethod` identifies the method that is starting point of the Workflow Function Execution. The [Workflow Execution](/docs/concepts/what-is-a-workflow-execution) completes when this method completes.
+A Workflow Definition interface in Java can only have one method annotated with `@WorkflowMethod`. It can be used to denote the [Workflow Type](/docs/concepts/what-is-a-workflow-type).
```

Comment:
```suggestion
A Workflow Definition interface in Java can have only one method annotated with `@WorkflowMethod`. It can be used to denote the [Workflow Type](/docs/concepts/what-is-a-workflow-type).
```